WordPress/wordpress-playground
# Website: Setup postMessage relay in the website app

Sets up the postMessage relay not only in the remote.html app, but also on the Playground website (index.html). This enables embedding it in an iframe and still passing messages to the nested WordPress.

This PR also moves the setupPostMessageRelay() function to the `@php-wasm/web` package

 ## Testing instructions

Add an `<iframe src="http://localhost:5400/website-server/"></iframe>`, load a WordPress plugin in it that adds a `message` event listener and logs it, post a message to the iframe that contains `{"type": "relay"}`, confirm it logs in the console.
